www/js/components/pages/vm/disks-form.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2019 Joyent, Inc.
+ */
+
+'use strict';
+/* eslint-disable max-len */
+
+var Backbone = require('backbone');
+var _ = require('underscore');
+var app = require('../../../adminui');
+
+var JobProgressView = require('../../../views/job-progress');
+var Template = require('./disks-form.hbs');
+var ViewModel = Backbone.Model.extend({});
+var DisksFormView = Backbone.Marionette.ItemView.extend({
+    template: Template,
+    id: 'disks-form',
+    attributes: {
+        'class': 'modal'
+    },
+    events: {
+        'submit form': 'doDiskAction',
+        'click .add': 'doDiskAction',
+        'click .resize': 'doDiskAction'
+    },
+
+    initialize: function (options) {
+        options = options || {};
+
+        if (options.vm) {
+            this.vm = options.vm;
+        }
+
+        if (!options.model) {
+            this.model = new ViewModel({ size: 0 });
+        }
```

Comment:
I'm not sure if `this.model` is used? If it is used, then when `options.model` is set, should it be saved to `this.model`?

Reply:
Actually, the `this.model` is pre-set by React so the `this.model` line here is useless. Removed.